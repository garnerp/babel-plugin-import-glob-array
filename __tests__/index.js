const babel = require('babel-core')
const plugin = require('../')
const fs = require('fs')
const path = require('path')

it('works', () => {
  const { content, filename } = loadFixture('basic')
  const { code } = babel.transform(content, { plugins: [plugin], filename })
  expect(code).toMatchSnapshot()
})

function loadFixture(name) {
  const filename = path.join(__dirname, `fixtures/${name}.js`)
  return { content: fs.readFileSync(filename, 'utf8'), filename }
}
