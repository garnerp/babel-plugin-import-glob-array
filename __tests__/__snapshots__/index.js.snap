// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`works 1`] = `
"import md from 'marked';
import { dir as _iga } from './docs/bar.md';
import { dir as _iga2 } from './docs/foo.js';
const foo = ['_iga', '_iga2'];
import _iga3, { _importMeta as _iga4 } from './docs/bar.md';
import _iga5, { _importMeta as _iga6 } from './docs/foo.js';
const wow = ['_iga3', '_iga5'];
const meta = ['_iga4', '_iga6'];"
`;
